src/App.tsx

```diff
@@ -7,6 +7,7 @@ import ListPage from "./pages/listPage/listPage";
 import Layout from "./pages/layout/layout";
 import SinglePage from "./pages/singlePage/singlePage";
 import 'leaflet/dist/leaflet.css'
+import ProfilePage from "./pages/profilePage/profilePage";
```

Comment:
Try to use absolute paths. If the structure changes, you won't encounter any issues